logdna/resource_stream_exclusion.go

```diff
@@ -161,7 +161,8 @@ func resourceStreamExclusion() *schema.Resource {
 			},
 			"title": {
 				Type:     schema.TypeString,
-				Required: true,
+				Default:  nil,
```

Comment:
Do you need the nil on the provider if the API has a null default?

Reply:
I'm not 100% sure if it's necessary but I think we should set the provider default to `nil` too for consistency's sake